operator/deploy/crds/cassandra.datastax.com_cassandradatacenters_crd.yaml

```diff
@@ -6123,8 +6136,8 @@ spec:
                 type: object
               type: object
             superUserUpserted:
-              description: The timestamp at which CQL superuser credentials were last
-                upserted to the management API
+              description: The timestamp at which cassandra user credentials were
+                last upserted to the management API
```

Comment:
This is overloaded now. We use the same timestamp for both updating the standard superuser as well as any additional users specified on the CRD. I've not changed the field name for backwards compatibility. We could add a separate field for updates to the other users, but that seems unnecessary.

Reply:
I think you can reset the status bits for the user creation. Make whatever is ideal for you, and label the existing stuff deprecated.